sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2281,6 +2281,8 @@ private void SetNameFieldsIfNull()
         /// <remarks>
         /// A <see cref="RequestFailedException"/> will be thrown if
         /// a failure occurs.
+        /// A DirectoryNotFoundException will be thrown if the file
```

Comment:
Could you provide a link to the exception itself as well?

So replace `DirectoryNotFoundException` with 
```
<see href="https://learn.microsoft.com/dotnet/api/system.io.directorynotfoundexception">DirectoryNotFoundException </see>
```